src/app/setup-wizard/setup-wizard-help-panel/energy-equipment-help/energy-equipment-help.component.html

```diff
@@ -1,4 +1,48 @@
 <p>
-    Add equipment to the "Industrial System Inventory" for the selected facility.
+    Add equipment to the "Industrial System Inventory" for the selected facility.</p>
+    <hr>
+<p>
+    This inventory focuses on equipment related to assessments that are being conducted: larger motor driven systems
+    (pumps, fans, compressors), process heating equipment, steam equipment. Systems like lighting equipment, HVAC
+    systems, and mobile equipment can also be tracked.
+</p>
+<hr>
+<p>
+    Setting the <span class="bold">"Industrial System"</span> sets the energy-related fields in the form.
+</p>
+<hr>
+<p>
+    <span class="bold">Utility Type</span> sets the units available for <span class="bold">Total
+        Size</span>.
+</p>
+<hr>
+<p>
+    Enter nameplate data for <span class="bold">Total Size</span> to begin estimating the energy used by the system.
+</p>
+<hr>
+<p>
+    <span class="bold">Operating Hours</span> refers to the hours the equipment (not the entire facility) is operating.
+</p>
+<hr>
+<p>
+    <span class="bold">Load Factor</span> refers to the fraction of the nameplate size power that is being used.
+</p>
+<hr>
+<p>
+    <span class="bold">Efficiency</span> refers to the energy efficiency of the equipment.
+</p>
+<hr>
+<p>
+    <span class="bold">Number of Equipment</span> can be utilized if multiple equipment is about the same size
```

Comment:
could add a period at the end (to be consistent with others.

Reply:
fixed